test.tf

```diff
@@ -108,24 +123,99 @@ resource "azurerm_postgresql_server" "example" {
   sku_name                     = "B_Gen5_2"
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure a firewall is attached to Azure SQL Server**
 
**Vulnerability**
Azure SQL Server does not have a firewall rule associated, this may make it too vulnerable.
 
**Remediation**
Data is the most important resource for any organization, firewalls implement access control, auditing, input validation and other security measures, therefor a firewall must always be implemented to keep SQL Severs secure.

source: accurics-beta